### PR TITLE
improvement: ZENKO-432 update to nodejs 8 lts

### PR DESCRIPTION
**Pull request by rahulreddy**

### CODE REVIEW

**Comment by rahulreddy** [comment]
@bert-e reset

**Comment by rahulreddy** [comment]
@bert-e wait

**Review by rachedbenmustapha — APPROVED** [review]

File: eve/main.yml [hunk 1]
```diff
@@ -132,6 +132,9 @@ stages:
         - '/home/eve/workspace'
     steps:
       - Git: *clone
+      - ShellCommand:
+          name: keyscan
+          command: ssh-keyscan -H github.com > ~/.ssh/known_hosts
```
Comment (ploki) [inline eve/main.yml]: related?
should ```mkdir -p ~/.ssh``` first

**Review by ploki — APPROVED** [review]

**Comment by tcarmet** [comment]
Hello, took a look into this PR and it's sounds like that there might be something I could fix/work on eve side to get it working properly (without workarounds and all) didn't had the time to work on it today, but I can take a look tomorrow. I'll let you guys decide (don't know if you're in a rush to get this merged or not.)

Cheers!

**Review by jonathan-gramain — APPROVED** [review]

File: eve/main.yml (hunk 1 above)
Comment (rahulreddy) [reply eve/main.yml]: I realized an easier way to get around this. Changes incoming.

**Comment by rahulreddy** [comment]
@bert-e wait

File: package.json [hunk 2]
```diff
@@ -64,7 +64,7 @@
     "tv4": "^1.2.7"
   },
   "optionalDependencies": {
-    "cdmiclient": "scality/cdmiclient#8f0c2e6"
+    "cdmiclient": "git+https://github.com/scality/cdmiclient#8f0c2e6"
```
Comment (tcarmet) [inline package.json]: Made the requested updates on eve to make it work with the `scality/cdmiclient#8f0c2e6` syntax.

Could you please make the change?

**Pull request closed without merging**